source/_components/deconz.markdown

```diff
@@ -0,0 +1,150 @@
+---
+layout: page
+title: "deCONZ"
+description: "Instructions on how to setup Conbee/Raspbee devices with deCONZ from Dresden Elektronik within Home Assistant."
+date: 2017-11-12 16:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: deconz.jpeg
+ha_category: Hub
+ha_release: "0.61"
+ha_iot_class: "Local Push"
+---
+
+[deCONZ](https://www.dresden-elektronik.de/funktechnik/products/software/pc/deconz/) by [Dresden Elektronik](https://www.dresden-elektronik.de) is a software that communicates with Conbee/Raspbee Zigbee gateways and exposes Zigbee devices that are connected to the gateway.
+
+[deCONZ REST API](http://dresden-elektronik.github.io/deconz-rest-doc/).
+
+Home Assistant will automatically discover deCONZ presence on your network, if `discovery:` is present in your `configuration.yaml` file.
+
+If you don't have the API key, you can generate an API key for deCONZ by using the one-click functionality similar to Philips Hue. Go to Menu->Settings->Unlock Gateway in deCONZ and then use the deCONZ configurator in Home Assistant GUI to create an API key. When you've generated the API key from Home Assistant, the API key will be stored in deconz.conf inside the home-assistant folder.
+
+You can add the following to your configuration.yaml file if you are not using the `discovery:` component:
+
+```yaml
+# Example configuration.yaml entry
+deconz:
+  host: IP ADDRESS
+```
+
+#### {% linkable_title Supported Device types %}
+
+- [Zigbee Lights](/components/light/deconz/)
+- [Humidity Sensors](/components/sensor/deconz/)
+- [Light Level Sensors](/components/sensor/deconz/)
+- [OpenClose Detectors](/components/binary_sensor/deconz/)
+- [Presence Detectors](/components/binary_sensor/deconz/)
+- [Pressure Sensors](/components/sensor/deconz/)
+- [Switches (Remote Controls)](/components/sensor/deconz/)
+- [Temperature Sensors](/components/sensor/deconz/)
+
+## {% linkable_title Configuration variables %}
+
+{% configuration %}
+host:
+  description: The IP address of your deCONZ web server.
+  required: false
+  type: string
+api_key:
+  description: The API key to access your deCONZ web server.
+  required: false
+  type: string
+port:
+  description: Configure port deCONZ web server is accessible from.
+  required: false
+  default: 80
+  type: int
+{% endconfiguration %}
+
+A full configuration could look like this:
+
+```yaml
+# Example configuration.yaml entry
+deconz:
+  host: 127.0.0.1
+  api_key: 0123456789
+  port: 80
+```
+
+## {% linkable_title Device services %}
+Available services: `configure`.
+
+#### {% linkable_title Service `deconz/configure` %}
+Set attribute of device in Deconz using [Rest API](http://dresden-elektronik.github.io/deconz-rest-doc/rest/).
+
+| Parameter | Description                                             |
+|-----------|---------------------------------------------------------|
+| `field`   | String representing a specific device in deCONZ.        |
```

Comment:
Please format as our standard service table. Also extend this table with info on if parameter is optional. See eg:
https://home-assistant.io/components/media_player.sonos/#service-sonos_snapshot